frontend/src/components/dashboard/aliases/Alias.tsx

```diff
@@ -236,6 +251,7 @@ export const Alias = (props: Props) => {
 type TooltipProps = {
   children: ReactNode;
 };
+
```

Comment:
nit: whitespace change 

Is this necessary?

Reply:
Removed whitespace change.